### ISSUE TITLE
Accordion (isOpenChange) doesnt work

### ISSUE THREAD

**Issue by alireza-saberi** [valor-software/ngx-bootstrap#3049]
### Bug description or feature request:
I am trying to change an icon based on one accordion being open or close. let's say they up/down arrow that I want to switch between these two. I tried [first example of accordion for this here:](https://valor-software.com/ngx-bootstrap/#/accordion) even by looking the console in demo for first example which is console.log the event nothing happens.
```import { Component } from '@angular/core';
 
@Component({
  selector: 'demo-accordion-basic',
  templateUrl: './basic.html'
})
export class DemoAccordionBasicComponent {
  log(event: boolean) {
    console.log(`Accordion has been ${event ? 'opened' : 'closed'}`);
  }
}```



### Versions of ngx-bootstrap, Angular, and Bootstrap:

ngx-bootstrap:^1.8.0
Angular: 4
Angular-cli: 1.4.4
Bootstrap:3.3

**Comment by IlyaSurmay** [comment]
![accordion](https://user-images.githubusercontent.com/18414330/32942835-e5b5d92a-cb92-11e7-9a28-2792d3866c5c.gif)
In the demo, the only third panel of accordion has the handler attached to `(isOpenChange)`, that's why it may seem that nothing's happening. Toggle third panel in the demo and you'll see that console.log works just fine. Otherwise, please provide a plunkr/stackblitz with reproduction.
You can use one of starter templates:
Plunkr: https://plnkr.co/edit/0NipkZrnckZZROAcnjzB?p=preview
StackBlitz: https://stackblitz.com/edit/ngx-bootstrap?file=app%2Fapp.module.ts

**Comment by IlyaSurmay** [comment]
Closed as can't reproduce (v2.0.0-beta.8), will be reopened if we have sample to reproduce

**Comment by umapyla** [comment]
@alireza-saberi - We have faced the same issue. isOpenChange won't work with version 1.8.0
so update your ngx-bootstrap and give a try.
